### PR TITLE
Tickets/dm 7451

### PR DESCRIPTION
**Pull request by mtpatter**
Add first sample alert and schema.

### CODE REVIEW

File: schema/diasource.avsc [hunk 1]
```diff
@@ -0,0 +1,176 @@
...
+			"name": "psCov", "fields": [
+				{"name": "psCov01", "type": "float"},
+				{"name": "psCov02", "type": "float"},
+				{"name": "psCov03", "type": "float"},
+				{"name": "psCov04", "type": "float"},
+				{"name": "psCov05", "type": "float"},
+				{"name": "psCov06", "type": "float"}
+					]},
+			"null"]},
+		{"name": "psFluxSigma", "type": ["float", "null"]},
```
Comment (danielsf) [inline schema/diasource.avsc]: Wouldn't `psFluxSigma` be the `[0,0]` entry in `psCov`?

File: schema/diasource.avsc [hunk 2]
```diff
@@ -0,0 +1,176 @@
...
+				{"name": "trailCov09", "type": "float"},
+				{"name": "trailCov10", "type": "float"},
+				{"name": "trailCov11", "type": "float"},
+				{"name": "trailCov12", "type": "float"},
+				{"name": "trailCov13", "type": "float"},
+				{"name": "trailCov14", "type": "float"},
+				{"name": "trailCov15", "type": "float"}
+					]},
+			"null"]},
+		{"name": "trailFluxSigma", "type": ["float", "null"]},
```
Comment (danielsf) [inline schema/diasource.avsc]: I think that these parameters (`trailFluxSigma`, `trailLengthSigma`, etc.) are included in `trailCov`.  There are 5 `trail` parameters: `Flux`, `Ra`, `Dec`, `Length`, and `Angle`.  The covariance matrix for 5 parameters would have 15 unique elements, making up the 15 `float`s in `trailCov`.

File: schema/diasource.avsc [hunk 3]
```diff
@@ -0,0 +1,176 @@
...
+		{"name": "dipLnL", "type": ["float", "null"]},
+		{"name": "dipChi2", "type": ["float", "null"]},
+		{"name": "dipN", "type": ["int", "null"]},
+		{"name": "fpFlux", "aliases": ["totFlux"], "type": ["float", "null"]},
+		{"name": "fpFluxSigma", "aliases": ["totFluxErr"], "type": ["float", "null"]},
+		{"name": "diffFlux", "type": ["float", "null"]},
+		{"name": "diffFluxSigma", "type": ["float", "null"]},
+		{"name": "fpSky", "aliases": ["fpBkgd"], "type": ["float", "null"]},
+		{"name": "fpSkySigma", "aliases": ["fpBkgdErr"], "type": ["float", "null"]},
+		{"name": "E1", "type": ["float", "null"]},
```
Comment (danielsf) [inline schema/diasource.avsc]: Are E1 and E2 ellipticity parameters?  Does this make them redundant with Ixx, Iyy, and Ixy?

File: schema/ssobject.avsc [hunk 4]
```diff
@@ -0,0 +1,87 @@
...
+				{"name": "q", "type": ["double", "null"]},
+				{"name": "e", "type": ["double", "null"]},
+				{"name": "i", "type": ["double", "null"]},
+				{"name": "lan", "type": ["double", "null"]},
+				{"name": "aop", "type": ["double", "null"]},
+				{"name": "M", "type": ["double", "null"]},
+				{"name": "epoch", "type": ["double", "null"]}
+				]}, "null"]},
+		{"name": "oeCov", "type": [{"type": "record",
+			"name": "oeCov", "fields": [
```
Comment (danielsf) [inline schema/ssobject.avsc]: Why are there no diagonal elements in this covariance matrix (I know the DPDD says there are only 21 elements; is that an error in the DPDD)?

**Review by danielsf — COMMENTED** [review]
I think you got everything that is supposed to be in the schema.  I just have a few comments about parameters that appear to be redundant.

**Comment by mtpatter** [comment]
Thanks, I wrote out all the covariance matrix terms.  In the DPDD, the number of terms in the covariance matrix seems to not count the diagonal for both the dipCov matrix in the diaSource table and oeCov matrix in the ssObject table.  For the diagonal terms I changed those all to suffix "Var" to avoid further confusion.  I think this should be self-consistent and consistent with the DPDD naming conventions of Err vs Sigma now (assuming those are right in the DPDD).

File: schema/ssobject.avsc (hunk 4 above)
Comment (timj) [reply schema/ssobject.avsc]: Remember that DPDD is not canonical. It's an example of how things might be, not a promise that they will be. See the schemas in the `cat` package for the current standard (and I think that's out of date).

**Comment by timj** [comment]
Please suggest fixes to DPDD if you find problems. Please fix the `cat` schemas. Please worry about Err vs Sigma problems.

**Comment by mtpatter** [comment]
Yes, I filed ticket DM-8050 because the cat schemas are out of date and because I am worried about the Err vs Sigma problems.

**Comment by timj** [comment]
This may sound daft, but couldn't we generate all the JSON from the `cat` schema SQL code?

**Comment by mtpatter** [comment]
Yes probably, in the future, it would be good to build the .avsc schemas from the `cat` schemas.  At this point I'm only at the level of getting data into a format that makes sense and making sure we're not obviously forgetting something.

**Comment by danielsf** [comment]
Looks fine to merge

**Pull request merged**